### PR TITLE
v2.0.0 release

### PR DESCRIPTION
**Pull request by koikonom**
Pre release changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
-## 1.27.0 (UNRELEASED)
+## 2.0.0 (Unreleased)
+
+BREAKING CHANGES:
+* `resource/virtual_machine`: Moving forward this provider will only work with Terraform version v0.12 and later.
+* `resource/virtual_machine`: [Deprecated attribute `name`](https://github.com/hashicorp/terraform-provider-vsphere/blob/master/CHANGELOG.md#130-january-26-2018)has been removed from the disk subresource.
```
Comment (redeux) [inline CHANGELOG.md:5]: Should this be `disk` subresource ?

Comment (koikonom) [reply CHANGELOG.md]: good call!

**Review by aareet — APPROVED** [review]

**Comment by redeux** [comment]
On the readme we should probably update the requirements.  Right now it says 0.11 is deprecated but available.  That won't be true anymore. https://github.com/hashicorp/terraform-provider-vsphere#requirements

... and if you're going to touch the readme, maybe remove the trending contributors sections?  It's blank at the moment. Not a great look.

**Review by redeux — APPROVED** [review]
Looks great, thanks!

**Pull request merged**